internal/livelint/livelint.go

```diff
@@ -8,18 +8,18 @@ import (
 )
 
 type Livelint struct {
-	k8s    kubernetes.Interface
+	K8s    kubernetes.Interface
 	config *rest.Config
-	http   *http.Client
```

Comment:
Why export it?

Reply:
So that tests can be moved to the test package. The exported functionality is not testable and currently we can either export arguably too much functionality OR have the tests in the test package, but not both. I prefer having a test package and then cleaning up the exported interface.

Larger refactoring being done, so I'd rather not spend too much time on details of something that is not correct.